flink-scala/src/main/scala/org/apache/flink/api/scala/DataSet.scala

```diff
@@ -1599,7 +1601,77 @@ class DataSet[T: ClassTag](set: JavaDataSet[T]) {
   def output(outputFormat: OutputFormat[T]): DataSink[T] = {
     javaSet.output(outputFormat)
   }
-  
+
+  /**
+    * Selects an element with minimum value.
+    * <p>
```

Comment:
This class does not use HTML for mark-up in comment. Please follow the style of the class.